value_test.go

```diff
@@ -531,14 +531,15 @@ func TestPersistLFDiscardStats(t *testing.T) {
 		persistedMap[fid] = val
 	})
 
+	db.vlog.discardStats.Unlock()
```

Comment:
I find it odd that we were acquiring the lock before and not calling unlock. Could you elaborate more on this?

Reply:
I also found that very odd.  I have no insight into why it was like this.  Probably an oversight.

Reply:
Seems like `db` variable is re-assigned and hence calling `Lock` again without `Unlock` is not a problem. But this is better.